aiohttp/helpers.py

```diff
@@ -31,6 +32,41 @@ def __new__(cls, login, password='', encoding='latin1'):
 
         return super().__new__(cls, login, password, encoding)
 
+    @classmethod
+    def decode(cls, auth_header, encoding='latin1'):
+        """Create a :class:`BasicAuth` object from an ``Authorization`` HTTP
+        header.
+
+        :param auth_header:  The value of the ``Authorization`` header.
+        :type auth_header:  str
+        :param encoding:  The character encoding used on the password.
+        :type encoding:  str
+
+        :returns:  The decoded authentication.
+        :rtype:  :class:`BasicAuth`
+
+        :raises ValueError:  if the headers are unable to be decoded.
+
+        """
+        split = auth_header.strip().split(' ')
+        if len(split) == 2:
+            if split[0].strip().lower() != 'basic':
+                raise ValueError('Unknown authorization method %s' % split[0])
+            to_decode = split[1]
+        elif len(split) == 1:
+            to_decode = split[0]
```

Comment:
I would be better strict to the specification where auth scheme is mandatory. Unlikely `Authorization: Zm9vOmJhcg==` will be a valid basic auth header.